plugins/ceph/check-ceph-mds.rb

```diff
@@ -0,0 +1,172 @@
+#! /usr/bin/env ruby
+#
+# check-ceph-mds
+#
+# DESCRIPTION:
+#   #YELLOW
+#
+# OUTPUT:
+#   plain text
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#   ceph client
+#
+# USAGE:
+#   #YELLOW
+#
+# NOTES:
+#   Runs 'ceph mds stat' command(s) to report health status of
+#   MDS ceph daemons. May need read access to ceph keyring and/or root access
+#   for authentication.
+#
+#   Using -u (--user) option allows to set the username to connect with
+#   the ceph cluster. By default the user is admin.
+#
+#   Using -c (--cluster) option allows to set the cluster name to connect with
+#   the ceph cluster. By default the user is ceph.
+#
+#   Using -m (--monitor) option allows to set an option monitor IP address to
+#   connect with the ceph cluster.
+#
+#   Using -t (--timeout) option allows to set the timeout in seconds to
+#   execute the ceph commands. By default the timeout is 10 seconds.
+#
+#   Using --show_stderr option allows to be considered the standard error
+#   output when executing the check. By default this option is false.
+#
+# LICENSE:
+#   Copyright 2013 Brian Clark <brian.clark@cloudapt.com>
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'rubygems' if RUBY_VERSION < '1.9.0'
+require 'sensu-plugin/check/cli'
+require 'timeout'
+require 'English'
+require 'json'
+
+class CheckCephMDSHealth < Sensu::Plugin::Check::CLI
+  option :user,
+         description: 'Client name for authentication',
+         short: '-u USER',
+         long: '--user',
+         proc: proc { |u| " --user #{u}" }
+
+  option :monitor,
+         description: 'Optional monitor IP',
+         short: '-m MON',
+         long: '--monitor',
+         proc: proc { |m| " -m #{m}" }
+
+  option :cluster,
+         description: 'Optional cluster name',
+         short: '-c NAME',
+         long: '--cluster',
+         proc: proc { |c| " --cluster=#{c}" }
+
+  option :timeout,
+         description: 'Timeout (default 10)',
+         short: '-t SEC',
+         long: '--timeout',
+         proc: proc(&:to_i),
+         default: 10
+
+  option :warn,
+         short: '-w WARN',
+         long: '--warn WARN',
+         description: 'Warn if the number of MDS daemons running are less than WARN',
+         proc: proc(&:to_f),
+         default: 2
+
+  option :crit,
+         short: '-c',
+         long: '--critical CRIT',
```

Comment:
These option definitions are copied from the original check and I didn't realized those inconsistencies.
I'll update this PR with those changes.